src/redist/redist.csproj

```diff
@@ -48,29 +48,37 @@
   <Target Name="PublishVersionFile"
           BeforeTargets="Publish">
 
-    <WriteLinesToFile File="$(SdkOutputDirectory)/.version"
+    <WriteLinesToFile File="$(PublishDir)/.version"
                       Lines="$(CommitHash);$(SdkVersion);$(Rid)"
                       Overwrite="true" />
   </Target>
 
+  <Target Name="PublishRoslyn"
+          BeforeTargets="Publish">
+          
+    <DotNetPublish ToolPath="$(Stage0Directory)"
```

Comment:
Should this be here now? Or should this just be a dependency between redist and tool_roslyn?